build_tools/cmake/iree_cc_test.cmake

```diff
@@ -112,7 +112,7 @@ function(iree_cc_test)
   set_property(TARGET ${_NAME} PROPERTY DIRECT_DEPS ${_RULE_DEPS})
 
   string(REPLACE "::" "/" _PACKAGE_PATH ${_PACKAGE_NS})
-  set(_TEST_NAME "${_PACKAGE_PATH}:${_RULE_NAME}")
+  set(_TEST_NAME "${_PACKAGE_PATH}/${_RULE_NAME}")
```

Comment:
I scanned our docs and build scripts just in case and didn't find any cases this renaming will break. Might have missed something though.